src/_data/pkg-vers.json

```diff
@@ -18,7 +19,8 @@
   "angular_router": {
     "doc-path": "guide/router",
     "prev-vers": "1.0.2",
-    "vers": "2.0.0-alpha+22"
+    "vers": "2.0.0-alpha+22",
+    "doc_vers": "2.0.0"
```

Comment:
You've added `docs_vers` but aren't using it, which isn't an issue since you don't need this new field. Use `vers` instead, and update it's value if necessary.